### DISCUSSION TITLE
Domain does not solve to the Github Pages Server

### DISCUSSION THREAD

**Discussion by Martiiim** [community/community#108530] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I was trying to configure my custom domain into a website of mine and i get this error: **NotServedByPagesError**, here how i configured the dns: 
<img width="802" alt="Captura de ecrã 2024-02-16, às 16 48 50" src="https://github.com/community/community/assets/116751609/aec6c932-9cdd-4c68-9689-e3593a186d6c">
Could anyone please help me?

**Comment by yoannchaudet** [comment]
For what custom domain you get an error? You have multiple stuff configured visible on this picture.

**Reply by Martiiim** [reply]
my custom domain is staysalmar.com

**Reply by Martiiim** [reply]
also i changed the dns config because ive seen some other dns records and they are different that the one i was using 
<img width="561" alt="Captura de ecrã 2024-02-16, às 17 26 39" src="https://github.com/community/community/assets/116751609/3e32c391-c11d-4ba7-ac76-f94d5596f923">

**Reply by yoannchaudet** [reply]
Your configuration is looking better now.

Remove these though, they are redundant and you don't need them (your `CNAME` for `www` covers it already):

<img width="1050" alt="image" src="https://github.com/community/community/assets/14911070/42fda7af-b4fd-4edb-8efb-084ed0625e8c">

If you just did changes you may have to wait a little for cache invalidation. Come back to your Pages settings and refresh the page later. It will try to re-validate your DNS to request a TLS certificate.

**Reply by Martiiim** [reply]
ok! just a question if i add www to the custom domain will it affect anything?

**Reply by yoannchaudet** [reply]
You will get a TLS certificate that covers both, staysalmar.com and www.staysalmar.com. It's a good practice to set them both.
